nuxt.config.js

```diff
@@ -1,23 +1,43 @@
 export default defineNuxtConfig({
-  runtimeConfig: {
-    public: {
-      API_URL:
-        process.env.NODE_ENV === "production"
-          ? "api.geode-solutions.com"
-          : "localhost",
+  $production: {
+    runtimeConfig: {
+      public: {
+        API_URL: "api.geode-solutions.com",
+        BASE_URL: "./public/",
+      },
+    },
+    devtools: {
+      enabled: false,
+    },
+  },
+
+  $development: {
+    runtimeConfig: {
+      public: {
+        API_URL: "localhost",
+        BASE_URL: "./public/",
+      },
+    },
+    devtools: {
+      enabled: true,
     },
   },
+
   modules: ["nuxt-electron", "vuetify-nuxt-module"],
   electron: {
     build: [
       {
-        // Main-Process entry file of the Electron App.
         entry: "electron/main.js",
       },
     ],
     renderer: {},
   },
 
+  router: {
+    options: {
+      hashMode: true,
```

Comment:
why?

Reply:
It return 404 when packaged if I remove this